examples/flows_and_runs_tutorial.py

```diff
@@ -83,9 +84,9 @@
 # When you need to handle 'dirty' data, build pipelines to model then automatically.
 task = openml.tasks.get_task(115)
 pipe = pipeline.Pipeline(steps=[
-    ('Imputer', preprocessing.Imputer(strategy='median')),
+    ('Imputer', impute.SimpleImputer(strategy='median')),
     ('OneHotEncoder', preprocessing.OneHotEncoder(sparse=False, handle_unknown='ignore')),
```

Comment:
Wouldn't it make sense to use the compose module here, too?

Reply:
With the reminder than I'm not an expert, I believe `ColumnTransformer` only makes sense when you need to do different transformations on different fields.  The previous example only wanted to do the OneHotEncoding on some of the columns, not all of them.   This example is apparently doing the OneHotEncoding on all(?) of the columns.

I suppose as an example of using Pipelines, it could probably go either way.  For the actual data being used in this example, it might make more sense to use a ColumnTransformer to only OneHotEncode the fields that are actually categorical (i.e. just the `attribute 'class' { tested_negative, tested_positive}` field) ?

Reply:
Makes sense, thanks a lot for the explanation